### PR TITLE
Add best effort support for locating WinSDK tools

### PR DESCRIPTION
**Pull request by spbrogan**
For platforms that don't have VS2017 or newer installed the vswhere based solution for locating the WinSDK dir doesn't work.  This change adds support for a best effort to look in known default install directories for the WinSDK tools.   The prefered method is still to use VsWhere and the vsvars files but this can help on older platforms.

### CODE REVIEW

**Comment by spbrogan** [comment]
Resolves Issue #2

File: edk2toollib/windows/locate_tools.py [hunk 1]
```diff
@@ -135,11 +137,16 @@ def FindWithVsWhere(products="*"):
...
+    if product is None:
+        product = "*"
+    if arch is None:
+        # TODO: look up host architecture?
+        arch = "amd64"
     interesting = set(keys)
     result = {}
-    ret, vs_path = FindWithVsWhere()
+    ret, vs_path = FindWithVsWhere(product)
     if ret != 0:
```
Comment (matthewfcarlson) [inline edk2toollib/windows/locate_tools.py]: You need to check if vs_path is not none. It looks like we don't return a non zero ret in all cases

Comment (spbrogan) [reply edk2toollib/windows/locate_tools.py]: done

**Comment by ericjin** [comment]
This change makes the solution friendly to user who don't install VS2017 or later (just try on one desktop with VS2015), give the hint "We didn't find VS path or otherwise failed to invoke vsWhere".

**Pull request merged**